Open Judge System/Web/OJS.Web/ViewModels/Common/DropdownViewModel.cs

```diff
@@ -64,7 +64,7 @@ public static IEnumerable<DropdownViewModel> GetYearsDropDownList(int fromYear,
         public static IEnumerable<DropdownViewModel> GetFromRange(int start, int end)
         {
             return Enumerable
-                .Range(start, end - start)
+                .Range(start, end)
```

Comment:
Modify to (start, end - start + 1) instead, so you can handle cases where the start is not one. Alternatively change the signature to be with parameters "start" and "count" so it will reflect Enumerable.Range correctly.